src/components/MyAccount/Settings/PasswordForm.tsx

```diff
@@ -221,20 +240,25 @@ const PasswordForm = ({ patronData, settingsState }: PasswordFormProps) => {
           <Flex>
             <Text
               sx={{
-                width: { base: "l", sm: "250px" },
+                width: { base: "200px", sm: "250px" },
                 marginTop: "xs",
-                marginLeft: { base: "l", lg: "unset" },
+                marginLeft: { base: "m", lg: "unset" },
                 marginBottom: 0,
               }}
             >
               ****
             </Text>
             {editingField === "" && (
               <EditButton
+                ref={focusRef as React.Ref<HTMLButtonElement>}
+                buttonLabel="Edit password"
                 buttonId="edit-password-button"
                 onClick={() => {
                   setIsEditing(true)
                   setEditingField("password")
+                  setTimeout(() => {
```

Comment:
Vera said: Does this shift the focus to the text input? If so, there actually should be two different refs, just to make it explicit what is being focused on.

Reply:
@EdwinGuzman this is what we discussed– basically now I'm using 1 ref for everything (except in `SettingsInputForm`, which uses an array of refs to match the array of inputs, and can't really be combined with a button ref). This makes it confusing to predict where the focus will go just from reading the code, but it does work, so I'm torn.

Reply:
For readability and debugging where focus goes, it seems like having two refs is the approach to follow. When reading `useRef<HTMLButtonElement | TextInputRefType | null>`, I was thinking that ref is doing too many things. I think we discussed one ref where possible and updating the `ref.current` value but that's not the case here.

Reply:
To be clear, what other problems are there?

Reply:
I think the issue is just conflicting directives. But I think me and Edwin agree this is a case where we can opt for readability.

Reply:
All refs now split up for readability again 👍